docs/source/api/deliveryservices_hostname_name_sslkeys.rst

```diff
@@ -72,6 +72,10 @@ Response Structure
 :state:           An optional field which, if present, contains the state entered by the user when generating certificate\ [#optional]_
 :version:         The version of the certificate record in Traffic Vault
 
+	.. versionadded:: 1.4
+
+:expiration:      The expiration date of the certificate for the :term:`Delivery Service`
+
 .. code- block:: http
```

Comment:
I know you didn't break this, but both the request example and response example on this page are broken, and it'd be great if you could fix them, if only because you're modifying one of them. The request example isn't indented properly to put it under the `code-block` directive, and this response directive has a space between the hyphen and `block` that's causing it to render as a block comment (i.e. not at all).

Reply:
done